lib/evaluate/host.cc

```diff
@@ -43,9 +44,7 @@ void HostFloatingPointEnvironment::SetUpHostFloatingPointEnvironment(
     currentFenv_.__mxcsr &= ~0x0040;  // operands
   }
 #else
-  // TODO other architectures
-  context.messages().Say(
-      "TODO: flushing mode for subnormals is not set for this host architecture when folding with host runtime functions"_en_US);
+  // Software flushing will be performed around host library calls if needed.
```

Comment:
I guess it's a placeholder for the warnings.

Reply:
What do you mean. Would you go for a #error in for unknown architectures/libc? I was planning on letting software flushing be the default then, but that is maybe not conservative enough.

Reply:
ah, ok, I see what you're up to.